.travis.yml

```diff
@@ -4,7 +4,8 @@ sudo: required
 branches:
   only:
     - master
-    - auto
```

Comment:
bors don't use the `auto` branch anymore?

Reply:
Homu used `auto`, Bors-ng uses `staging`. I'm not exactly sure why. :)